### ISSUE TITLE
Ctlptl stops waiting for Docker Desktop to be ready before Kubernetes is actually ready

### ISSUE THREAD

**Issue by TimothyLoyer** [tilt-dev/ctlptl#131]
This error occurs while applying a cluster config from YAML on Windows or Mac. It may be of note that ctlptl does not wait the full 2 minutes, and if I repeat the apply after the Kubernetes icon goes green in Docker Desktop, it will complete without error.
```
ctlptl apply -f cluster.yaml
Applied new Docker Desktop settings. Waiting 2m for Docker Desktop to restart...
Switched to context "docker-desktop".
configuring cluster: Delete "https://kubernetes.docker.internal:6443/api/v1/namespaces/kube-public/configmaps/ctlptl-cluster-spec": EOF
Setup could not complete due to an error.
```

Here are the contents of my cluster.yaml:
```yaml
apiVersion: ctlptl.dev/v1alpha1
kind: Cluster
name: docker-desktop
product: docker-desktop
```

This was run via Windows Terminal + Powershell (5.1), Docker Desktop (4.0.1) with WSL2 on Windows 10, and also from bash on Mac OS 11.6, also with Docker Desktop 4.0.1.

(Edit: Originally I thought this was only an issue on Windows, but now that I have tested it on Mac OS as well (with the help of a co-worker) and I am seeing the same issue on Docker Desktop no matter what.)

**Comment by nicks** [comment]
thanks for the report! Ya, there's been pretty heavy development on docker-desktop recently and i wonder if they changed the order of operations. probably just needs to wait a little longer for it to create a k8s context.

**Comment by nicks** [comment]
this should be fixed in v0.5.6
